lib/fetch/sanitize.js

```diff
@@ -149,7 +149,7 @@ export function sanitizeBadger(json) {
     imageUrl: get('badger.image-url'),
     startDate: get('badger.start-date'),
     about: get('badger.about'),
-    skills: (get('badger.skills') || []).map(data => data.skill.value),
+    skills: (get('badger.skills') || []).filter(data => data.skill).map(data => data.skill.value),
```

Comment:
Skills in Prismic are a modelled as a Group of inputs. So, you can add a new item to the Group but not set any text. This appears as an empty item. (It causes the fetcher to fall over)